src/main/java/minicraft/screen/WorldGenDisplay.java

```diff
@@ -127,19 +131,40 @@ public String getUserInput() {
 				return super.getUserInput().toLowerCase(Localization.getSelectedLocale());
 			}
 
+			private int maxXDisplacement = 0; // Relative maximum rendering displacement.
+			private int renderingX = 0; // Relative rendering displacement.
+
+			{
+				setChangeListener(i -> renderingX = maxXDisplacement = getUserInput().length() > 9 ? -(getUserInput().length()-9) * 8 : 0);
+			}
```

Comment:
Using these brackets means you have designed bad code. This should be done in the constructor or something.

Reply:
Uhh isn’t a constructor a method with the same name as the class? There is clearly a “public worldGenDisplay()” method.

Reply:
This is not how an anonymous class works.

Reply:
> Since they have no name, we can't extend them. For the same reason, anonymous classes cannot have explicitly declared constructors.

Source: https://www.baeldung.com/java-anonymous-classes#1-constructor

Reply:
Stop avoiding the question. You know what I mean. Put it in the constructor.

Reply:
That is an anonymous class.

Reply:
You mean an inner class? Those can have constructors.